sys_conn_oob.go

```diff
@@ -229,7 +229,14 @@ func (c *oobConn) ReadPacket() (receivedPacket, error) {
 
 // WritePacket writes a new packet.
 // If the connection supports GSO, it's the caller's responsibility to append the right control mesage.
-func (c *oobConn) WritePacket(b []byte, addr net.Addr, oob []byte) (int, error) {
+func (c *oobConn) WritePacket(b []byte, addr net.Addr, packetInfoOOB []byte, gsoSize uint16) (int, error) {
+	oob := packetInfoOOB
```

Comment:
Do you need this declaration? Is is possible to just use packetInfoOOB since it is local anyway?

Reply:
I don't, but maybe it's cleaner? I want to make it clear that the caller should only pass in the packet info control message.

Reply:
sure. That's fine by me.